tests/setup_module_utils.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+if [ ! -d $1 -o ! -d $2 ] ; then
+    exit 0
+fi
+
+# we need absolute path for $2
+absmoddir=$( cd $2 ; pwd )
```

Comment:
you could use `readlink -f "$2"` instead

Reply:
used `realpath` here

Reply:
I am afraid `realpath` is not available on Ununtu images (or was not available, the situation may be different now). `readlink -f "$2"` is a good alternative.